setup.py

```diff
@@ -3,14 +3,14 @@
 from setuptools import setup
 
 setup(name='tap-mongodb',
-      version='3.2.0',
+      version='3.3.0',
       description='Singer.io tap for extracting data from MongoDB',
       author='Stitch',
       url='https://singer.io',
       classifiers=['Programming Language :: Python :: 3 :: Only'],
       py_modules=['tap_mongodb'],
       install_requires=[
-          'singer-python==6.0.1',
+          'singer-python @ git+https://github.com/singer-io/singer-python.git@SUP-6525-allow-nan',
```

Comment:
This is good for PR alpha, but cannot be released in prod pinning to a git branch, any further installs will fail if the branch gets deleted

Reply:
Please Release the singer-python first and then update the PR